charts/celeborn/values.yaml

```diff
@@ -42,23 +42,29 @@ securityContext:
 # - emptyDir
 # - hostPath
 # Note: size only works in emptyDir type
+# mountPath works in hostPath type using to set `volumes hostPath path`
 # Celeborn Master will pick first volumes for store raft log
 volumes:
   master:
     - path: /mnt/rss_ratis
+      volumePath: /mnt/rss_ratis
       type: hostPath
       size: 1Gi
   worker:
     - path: /mnt/disk1
+      volumePath: /mnt/rss_ratis
```

Comment:
LGTM, we may can change volumePath: /mnt/rss_ratis to another name such as /mnt/volume.

Reply:
can we rename use `mountPath` and `hostPath`?

Reply:
Sure, fixed.